test/osx_linux_tests.sh

```diff
@@ -0,0 +1,32 @@
+PY_GENERATE="python2.7"
+PY_TEST="python2.7"
+
+while getopts g:t: option 
```

Comment:
Can we make these arguments more explicit? Instead of g and t, something like "generated_python_version" and "test_python_version"?